src/XCCDF_POLICY/xccdf_policy.c

```diff
@@ -2060,6 +2060,11 @@ struct xccdf_score * xccdf_policy_get_score(struct xccdf_policy * policy, struct
 	return xccdf_result_calculate_score(test_result, (struct xccdf_item *) benchmark, scsystem);
 }
 
+int xccdf_policy_update_score(struct xccdf_policy * policy, struct xccdf_result * test_result)
```

Comment:
Since the wrapper around `xccdf_result_calculate_score` was called `xccdf_policy_get_score`, I was going for something that follows a similar logic. :-)

Nevertheless, I will make this change.